R/ResampleDesc.R

```diff
@@ -141,6 +149,53 @@ makeResampleDescRepCV = function(reps = 10L, folds = 10L) {
   makeResampleDescInternal("repeated cross-validation", iters = folds * reps, folds = folds, reps = reps)
 }
 
+makeResampleDescFixedCV = function(horizon = 1L, initial.window = .5, size = 100L, skip = .01  ) {
+  horizon = asInteger(horizon, lower = 1L, upper = Inf)
+  assertNumeric(initial.window, lower = 1E-15, upper = 1)
+  size = asInteger(size, lower = 1L, upper = Inf)
+  assertNumeric(skip, lower = 0L, upper = Inf)
+  initial.window.desc = floor(initial.window * size)
+  skip.desc = floor(skip * size)
+  stops = seq(from = 1, to = size)[initial.window.desc:I(size - horizon)]
+  starts = stops - initial.window.desc + 1
+  thin = function(x, skip = 2) {
```

Comment:
I have changed these to `horizon = 1L, initial.window = .5, skip = 0`

Because one step ahead forecasting is the most common, we have to pick some initial window for default so half the data is pretty okay, and skip is not necessary.